Logging configuration for specific declarative RestClient using named "client" properties aka quarkus.rest-client."client".logging.* (#45520)
quarkusio/quarkus: docs/src/main/asciidoc/rest-client.adoc

NOTE: REST Client is logging the traffic with level DEBUG and does not alter logger properties. You may need to adjust your logger configuration to use this feature.

These configuration properties work globally for all clients injected by CDI.

> If you want configure logging for particular one (but only for CDI injected), you should do it by specifying named "client" properties, also known as `quarkus.rest-client."client".logging.*` properties.

Reviewer: ```suggestion
If you want configure logging for a specific declarative client, you should do it by specifying named "client" properties, also known as `quarkus.rest-client."client".logging.*` properties.
```

An example logging configuration:

[source,properties]
----
quarkus.rest-client.logging.scope=request-response
quarkus.rest-client.logging.body-limit=50

quarkus.rest-client.extensions-api.scope=all

quarkus.log.category."org.jboss.resteasy.reactive.client.logging".level=DEBUG

> quarkus.log.console.level=DEBUG

Reviewer: Is this needed?

Author: We often forget about this, butwithout it nothing will be output to the logs :D I thought it would be useful to give a clear example in the article

Reviewer: 👍🏽
